lib/gmail/lib.tsx

```
@@ -2,8 +2,9 @@ import { render } from "@react-email/render";
 import { and, desc, eq, isNotNull, isNull } from "drizzle-orm";
 import { htmlToText } from "html-to-text";
 import MailComposer from "nodemailer/lib/mail-composer";
+import { takeUniqueOrThrow } from "@/components/utils/arrays";
```

ESLint

[error] 'takeUniqueOrThrow' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)